AllReadyApp/Web-App/AllReady/DataAccess/AllReadyDataAccessEF7.Activity.cs

```diff
@@ -17,7 +17,7 @@ IEnumerable<Activity> IAllReadyDataAccess.Activities
                                 .Include(a => a.Location)
                                 .Include(a => a.Location.PostalCode)
                                 .Include(a => a.Campaign)
-                                .Include(a => a.Campaign.ManagingOrganization)
+                                .ThenInclude(c => c.ManagingOrganization)

```

Comment:
Can you indent this or include it on the previous line to make it clear that it's an extension of `Campaign` rather than `Activity`?